test/e2e/lib/gutenberg/tracking/general-tests.js

```diff
@@ -158,6 +158,22 @@ export function createGeneralTests( { it, editorType, postType } ) {
 		);
 	} );
 
+	it( `W button should not trigger the "wpcom_block_editor_close_click" event when it's connected to a sidebar`, async function () {
```

Comment:
Could you explain how this test ensures that the `W` button is not triggering this event? I don't see that button being pressed anywhere... 🤔

Reply:
Ah, scratch that. I thought is's about a `W` key, like in `WSAD` 😅. Anyway, what that `W` button actually is on the page? 😄

Reply:
Right! We don't really have an offical name for the button. Either W Button, WP Button or Site Icon Button

Reply:
Ah, makes sense! Although we did name it in the `toggleBlockEditorSidebar` as the `sidebarButton`, no? 😄 I think we can keep the "Block editor sidebar button" name for clarity, as it's what is displayed as the element's title:

<img width="142" alt="Screenshot 2021-06-24 at 15 51 51" src="https://user-images.githubusercontent.com/1451471/123275287-7ab1f300-d504-11eb-8c91-a83ff0967368.png">